src/debugger/debugAdapter.ts

```diff
@@ -24,18 +24,18 @@ import { SwiftToolchain } from "../toolchain/toolchain";
  * Class managing which debug adapter we are using. Will only setup lldb-vscode/lldb-dap if it is available.
  */
 export class DebugAdapter {
-    private static debugAdapaterExists = false;
-
     /** Debug adapter name */
     public static get adapterName(): string {
-        return configuration.debugger.useDebugAdapterFromToolchain && this.debugAdapaterExists
-            ? "swift-lldb"
-            : "lldb";
+        return "swift-lldb";
     }
 
     /** Return debug adapter for toolchain */
-    public static getDebugAdapter(swiftVersion: Version): "lldb-vscode" | "lldb-dap" {
-        return swiftVersion.isLessThan(new Version(6, 0, 0)) ? "lldb-vscode" : "lldb-dap";
+    public static getDebugAdapterType(swiftVersion: Version): "lldb-vscode" | "lldb-dap" {
+        return configuration.debugger.useCodeLLDB
+            ? "lldb-vscode"
```

Comment:
question: does CodeLLDB even allow specifying a debug adapter?

Reply:
suggestion: assuming it is to make other code work below then maybe we should just adjust that logic

Reply:
No, CodeLLDB does not allow specifying a debug adapter. This is used in other places to determine which debug adapter we're using.